com.github.babluboy.bookworm.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "com.github.babluboy.bookworm",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "bookworm",
+    "rename-appdata-file": "com.github.babluboy.bookworm.appdata.xml",
+    "finish-args": [
+            "--share=ipc",
+            "--socket=x11",
+            "--device=dri",
+            "--socket=wayland",
+            "--filesystem=home",
+            "--talk-name=ca.desrt.dconf",
+            "--filesystem=xdg-run/dconf",
+            "--filesystem=~/.config/dconf:ro",
+            "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+
+    "build-options": {
+        "cflags": "-O2",
+        "cxxflags": "-O2"
+    },
+    "modules": [
+         {
+                "name": "openjpeg",
+                "buildsystem" : "cmake",
+                "sources" : [
+                        {
+                                "type" : "git",
+                                "url" : "https://github.com/uclouvain/openjpeg"
```

Comment:
this needs to have a specific release tag or commit. Generally we find it's best to use archives like: https://github.com/uclouvain/openjpeg/releases

Reply:
Changed to a release archive and made a pull request. Thanks for pointing out :-)

Reply:
Great, when that's merged I'll rebuild.